nephio-ansible-install/roles/repo-gitea/create/tasks/main.yaml

```diff
@@ -0,0 +1,27 @@
+- name: Slurp access token
+  ansible.builtin.slurp:
+    src: /home/{{ cloud_user }}/gitea/nephio-gitea-token
+  register: gitea_token
+  when:
+    - gitea_username is defined
+- name: Create repos
```

Comment:
I see the below when this task is being executed

```
failed: [localhost] (item={'key': 'mgmt', 'value': {'mgmt_subnet': '172.88.0.0/16', 'pod_subnet': '10.196.0.0/16', 'svc_subnet': '10.96.0.0/16'}}) => {"ansible_loop_var": "item", "cache_control": "no-store, no-transform", "changed": false, "connection": "close", "content_length": "74", "content_type": "application/json;charset=utf-8", "date": "Tue, 13 Dec 2022 16:12:02 GMT", "elapsed": 0, "item": {"key": "mgmt", "value": {"mgmt_subnet": "172.88.0.0/16", "pod_subnet": "10.196.0.0/16", "svc_subnet": "10.96.0.0/16"}}, "json": {"message": "token is required", "url": "http://localhost:3000/api/swagger"}, "msg": "Status code was 401 and not [201]: HTTP Error 401: Unauthorized", "redirected": false, "status": 401, "url": "http://localhost:3000/api/v1/user/repos", "x_content_type_options": "nosniff", "x_frame_options": "SAMEORIGIN"}
```

Reply:
Hi @vjayaramrh , this is a follow on error from the issue you mentioned in your review comment above. Removing the -t in the docker exec will allow for that task to run which will unblock this one.

Reply:
@danaschwanden sounds good

Reply:
When I ran again on a newly created VM along with @henderiw , I did not encounter this issue again. So, can ignore the above comment for now